CMakeLists.txt

```diff
@@ -12,6 +12,16 @@ site_name(HOSTNAME)
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
 
+# First search for Python3 then fall back to Python2, since specifying
```

Comment:
put this in `cmake/SetupPython.cmake`  and also change `SpectreSetupPython` to `SetupSpectrePythonPackage`